src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -241,3 +239,11 @@
         line-height: calculateRem(36px);
     }
 }
+
+@mixin nav-item-active-hover {
```

Comment:
Please create a proper mixin file in the `mixins` folder and import it in this file.